### ISSUE TITLE
Modification of a read-only value attempted at /opt/traffic_ops/app/lib/UI/Topology.pm line 92.

### ISSUE THREAD

**Issue by dewrich** [Comcast/traffic_control#1222]
John Rushford pointed this issue out when he tried to Tools-> SnapshotCRConfig.  See attachment

**Comment by dewrich** [comment]
![screen shot 2016-03-25 at 11 07 59 am](https://cloud.githubusercontent.com/assets/56597/14049282/e46d1496-f279-11e5-8272-cdf5a20afab0.png)

**Comment by knutsel** [comment]
No Traffic Router defined in CDN, so snapshot CRConfig is moot.

**Comment by elsloo** [comment]
Also, note that this bug is not present in master; only 1.4.x. Master looks like this:

```
            my $t = ( $cachetype eq 'CCR' ) ? 'Traffic Router' : $cachetype;
            my $e = Mojo::Exception->throw( "No $t profiles found for CDN: " . $cdn_name );
```

**Comment by dewrich** [comment]
@elsloo  Great catch, looks like it was fixed in 1.5.0, I'll close this one.
